spec/requests/checkouts_spec.rb

```diff
@@ -118,4 +118,52 @@
       expect(response).to redirect_to(checkouts_url)
     end
   end
+
+  describe "PATCH /return" do
+    it "updates the return time of requested checkout" do
+      patch return_checkout_url(checkout)
+      checkout.reload
+      expect(checkout.return_time).to be <= DateTime.current
+    end
+    it "redirects to the checkouts list" do
+      patch return_checkout_url(checkout)
+      expect(response).to redirect_to(checkouts_url)
```

Comment:
Is there a way to redirect to the page a user is currently viewing? If I'm trying to return multiple, but not all, of my current checkouts it would be frustrating to have to keep going back to the checkouts page because I keep getting redirected.

Reply:
That's a great scenario!  Totally makes sense.
We haven't really used it (or not much), but there is a redirect back.  There is a new method for it in rails 6: `redirect_back`
https://api.rubyonrails.org/v6.0.5/classes/ActionController/Redirecting.html#method-i-redirect_back

Reply:
That looks good. Any thoughts on what would be the best `fallback_location`?

Reply:
How about the Checkouts page?

Reply:
That's what I was leaning towards. Will make the change.